### ISSUE TITLE
add skip link

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/data-explorer#85]
Add a skip link to the site. This can follow the exact same structure as the skip link in the Lotus app:

https://github.com/digitallinguistics/app/blob/main/src/App/Skip-Link/Skip-Link.hbs

This element should be placed immediately after the opening `<body>` tag, so that it's the first element in `<body>`. Use a Handlebars partial for it:

```hbs
{{> skip-link }}
```

This will just be HTML for now, not styling.
